easybuild/easyconfigs/p/pyiron/pyiron-0.5.1-foss-2023a.eb

```
@@ -0,0 +1,114 @@
+    ('Pint', '0.23'),
+    ('SQLAlchemy', '2.0.25'),
+]
+
+check_ldshared = True
+use_pip = True
+
+exts_list = [
+    ('h5io', '0.1.2', {
+        'checksums': ['6400543224c489f1cf487be551f892310fa7060fcd4935b84e515efdc1e1fa43'],
+    }),
+    ('defusedxml', '0.6.0', {
+        'checksums': ['f684034d135af4c6cbb949b8a4d2ed61634515257a67299e5f940fbaa34377f5'],
+    }),
+    ('pysqa', '0.1.7', {
+        'checksums': ['fdc37e0178649750e7386b4e9b8287693ee2d47559723c11b9ad42afcfc55900'],
+    }),
+    ('pyfiglet', '0.8.post1', {
+        'checksums': ['c6c2321755d09267b438ec7b936825a4910fec696292139e664ca8670e103639'],
+    }),
+    ('mendeleev', '0.14.0', {
+        'checksums': ['1a89ccf05c708aebe627c5eb19ff5c6379585a2f9e588e1dcf9b03182fe61f3c'],
+    }),
+    ('pyfileindex', '0.0.18', {
+        'checksums': ['6d5acf2500f568667aa5273ae57692c2b62facbcaae7c0bbbf21ba3bdd904961'],
+    }),
+    ('seekpath', '2.1.0', {
+        'checksums': ['31cec579628262e6d4a4c3693fefa70d6ccae1ceeef7c9d10ea3cd48988452c4'],
+    }),
+    (name, version, {
+        'modulename': False,
+        'checksums': ['2e5e3f892b7e49a15443569870aa4ea049951b71fe2ad1ad7f4ac7551d475788'],
+    }),
+    ('pyiron_base', '0.6.12', {
+        'patches': ['pyiron-0.5.1_fix-pyiron-base-version.patch'],
+        'source_urls': ['https://github.com/pyiron/pyiron_base/archive/refs/tags/'],
+        'checksums': [
+            {'pyiron_base-0.6.12.tar.gz': '7ef8fa8b21724776c1ced3ee1efb8484984234150a7f3da296577f90c63751cb'},
+            {'pyiron-0.5.1_fix-pyiron-base-version.patch':
+             'a6390dcd366e17361fb71f26c6d3b75835526bdf773d9ef535c02836fe551b31'},
+        ],
+    }),
+    ('atomistics', '0.1.15', {
+        'patches': ['pyiron-0.5.1_fix-atomistics-requirements.patch'],
+        'source_urls': ['https://github.com/pyiron/atomistics/archive/refs/tags/'],
+        'checksums': [
+            {'atomistics-0.1.15.tar.gz': 'e9c505cace0cbeb9ea5ed33799f332f5cc95230a05ba14b646475cff1bdecaee'},
+            {'pyiron-0.5.1_fix-atomistics-requirements.patch':
+             'deaa57c27be147efc7e4926f6795bfa421280a299fc96de8c0bae342e4770d6c'},
+        ],
+    }),
+    ('pyiron_atomistics', '0.3.11', {
+        'patches': ['pyiron-0.5.1_fix-pyiron-atomistics-version.patch'],
+        'source_urls': ['https://github.com/pyiron/pyiron_atomistics/archive/refs/tags/'],
+        'checksums': [
+            {'pyiron_atomistics-0.3.11.tar.gz': '8c73c452fe60cb808961d22361f9b749488e3d50a8bda7f5c0132aea2f69e00f'},
+            {'pyiron-0.5.1_fix-pyiron-atomistics-version.patch':
+             '5c9b987cb63508f482bff696e252987e10df9b8ee8e860491583c8e547d4568c'},
+        ],
+    }),
+    ('pylammpsmpi', '0.2.10', {
+        'patches': ['pyiron-0.5.1_fix-pylammpsmpi.patch'],
+        'source_urls': ['https://github.com/pyiron/pylammpsmpi/archive/refs/tags/'],
+        'checksums': [
+            {'pylammpsmpi-0.2.10.tar.gz': '0e74b7ccfdb5550308ac3462cd02d1fd30d99472552271d17da0b00ba754281c'},
```

```suggestion
            {%(name)s-%(version)s.tar.gz': 'bd5af29a935dacbee743c3cc0bcc799e24e7c2483801f56cb9092a79f016f2ed'},
```
This is what I get when I download the file...